t/admin/services.t

```diff
@@ -1085,3 +1085,50 @@ GET /t
 --- error_code: 400
 --- no_error_log
 [error]
+
+
+
+=== TEST 31: set service(id: 1)（allow empty `service` object）
```

Comment:
Insufficient test cases, can I add content in the empty object later?

Reply:
OK, I will add the complete test case.